src/tests/registration/constants.ts

```diff
@@ -0,0 +1,56 @@
+import QuestionType from "constants/QuestionType";
+import StudentRole from "constants/StudentRole";
+
+export enum TestValue {
```

Comment:
why an enum over a json object?

Reply:
kind of nice bc enums offer a bit more type safety & security that the value won't change! open to changing this in a PR if we have a good reason to (just wanna merge this speedy)

Reply:
no reason to change, was just wondering